files/cloud.cfg

```diff
@@ -0,0 +1,82 @@
+# WARNING: Modifications to this file may be overridden by files in
+# /etc/cloud/cloud.cfg.d
```

Comment:
is there an upstream we can add a reference to here, so we can rebase on the defaults every once in a while? did you just grab this off an AL2 instance?

seems like this might be the source of truth, but it's a jinja template 🤢

Reply:
Yeah I grabbed it off an instance. That template resolved isn't what is shipped in AL2. I asked and it looks like it is only available in the cloud-init RPM in the amzn2 yum repo. It's a little unfortunate, but I'm also told it's rarely updated.

Reply:
Ah bummer. I guess you already considered patching `cloud.cfg` instead of wholesale replacing it? Doesn't seem like there's a way we can disable this module with a `/etc/cloud/cloud.cfg.d` drop-in either

Reply:
Yeah, definitely went down that path first and it doesn't look like there's a way to do with with `/etc/cloud/cloud.cfg.d`.

Reply:
Changed it to just sed the file to remove that one module.